ESQL: Raise timeout on test suite

nik9000 opened the pull request:
It uses `semantic_text` which can be quite slow.

Closes #128513
Closes #128511

nik9000 committed:
ESQL: Raise timeout on test suite

It uses `semantic_text` which can be quite slow.

Closes #128513
Closes #128511

elasticsearchmachine commented:
Pinging @elastic/es-analytical-engine (Team:Analytics)

elasticsearchmachine commented:
## 💚 Backport successful
| Status | Branch | Result |
|:------:|:------:|:------:|
| ✅ |  [8.19](https://github.com/elastic/elasticsearch/pull/128528)  | [<img src="https://img.shields.io/github/pulls/detail/state/elastic/elasticsearch/128528">](https://github.com/elastic/elasticsearch/pull/128528) |

nik9000 commented:
The timeouts I'm seeing are Windows related. I added this in #128156.

nik9000 commented:
> I added this in #128156.

Which was late last week.



> The timeouts I'm seeing are Windows related.

And our Windows tests are notoriously slow.




One of the failures doesn't look like a timeout:
```
Caused by: java.lang.IllegalStateException: cluster state update executor did not preserve version numbers: [update model deployment]
```

I've added that to https://github.com/elastic/elasticsearch/issues/121165

nik9000 commented:
Hmmm, here's another failure:
```
Caused by: java.lang.IllegalStateException: cluster state update executor did not preserve version numbers: [update model deployment]
```

nik9000 commented:
I don't think this timeout bump really does anything to be honest. I was too eager to bump the timeout but it looks like ML was acting funky.

nik9000 commented:
@iverase, I opened https://github.com/elastic/elasticsearch/pull/128591 which replaces the timeout bump with a hack that speeds up this text field and skips the download. That *should* make these failures drop away. I'd love for there to be a way to automatically catch these things in tests, but, for now, this should help.
